src/winetricks

```diff
@@ -11971,6 +11971,67 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    w_package_unsupported_win64
+
+    if [ ! -x "$(which ntlm_auth 2>/dev/null)" ]; then
+        w_die "winbind (part of samba) is required for the installation"
+    fi
+
+    #link from https://www.askvg.com/direct-download-link-microsoft-office-2013-professional-plus-free-trial/
+    w_download http://care.dlservice.microsoft.com/dl/download/2/9/C/29CC45EF-4CDA-4710-9FB3-1489786570A1/OfficeProfessionalPlus_x86_en-us.img 236f8faae3f979ec72592a63784bba2f0d614916350c44631221b88ae9dae206 "OFFICE15.iso"
+
+    w_set_winver win7
+
+    w_call corefonts
+    w_call riched20
```

Comment:
There is a start-issue for PowerPoint using `wine-3.0`. It is starting the installer. Not pressing "reboot" works around this. I do not have these issues on my other pc. I tried to install the elsewhere reported requirements `msftedit riched30 vb6run` but it does not help. Can anybody confirm?

![screenshot from 2018-02-19 11-38-16](https://user-images.githubusercontent.com/16167358/36433644-44d42fdc-165d-11e8-9ea1-a991e251be9e.png)

The [log from wine](https://pastebin.com/NXrbj3iJ) and [backtrace](https://pastebin.com/gyhXfPnm).

Should I move this over to winehq or does anyone have a solution for it on winetricks?

Reply:
installed on a clean wine-3.0 prefix (unattended installation).
powerpoint now once tried to repair and afterwards complained on a missing license key. Now it stts perfectly.

Could it have been due to the DirectX workaround that is now dropped on wine<3

Reply:
1e68d494e54d8c1530413f9a3ca2935a63d26210